config/locales/en.yml

```diff
@@ -72,4 +72,28 @@ en:
             Schedule a maintenance window during which no events are streamed to BigQuery
             in the format of '22-01-2024 19:30..22-01-2024 20:30' (UTC).
           default: ENV['BIGQUERY_MAINTENANCE_WINDOW']
-
+        azure_federated_auth:
+          description: |
+            Whether to use azure workload identity federation for authentication
```

Comment:
@asatwal does this also switch between APIs? If so could we add that to the description?

Reply:
Yes this does switch APIs as the old APIs don't work with the new WIF Auth. Good point I'll add to this description.